### PR TITLE
[bug] Modo: report a message when there is an image

### PR DESCRIPTION
**Pull request by LeRoiDesKiwis**
I fixed the bug where we couldn't report a message when there is an image. Now it's showing "no content" in the "Message content" section.
image

### CODE REVIEW

File: module_modo.lua [hunk 1]
```diff
@@ -249,6 +249,10 @@ function Module:HandleEmojiAdd(userId, message)
 			content = content:sub(1, 800) .. "...<truncated>"
 		end
 
+		if (content ~= nil) then
```
Comment (SirLynix) [inline module_modo.lua:252]: The check is wrong, this is checking if there's content and overriding it if so.

Comment (SirLynix) [reply module_modo.lua:252]: Plus, this check is not required as it's already done on line 271
```lua
					value = content or "<empty>"
```
I suppose Discord rejects valid but empty messages (which could be checked with `(content and #content > 0) and content or "empty"`)

File: module_modo.lua [hunk 2]
```diff
@@ -245,6 +245,7 @@ function Module:HandleEmojiAdd(userId, message)
 		local reporterUser = client:getUser(userId)
 
 		local content = message.cleanContent
+
```
Comment (AntoineJT) [inline module_modo.lua:248]: Ajout de ligne inutile

File: module_modo.lua [hunk 3]
```diff
@@ -268,7 +269,7 @@ function Module:HandleEmojiAdd(userId, message)
 				},
 				{
 					name = "Message content",
-					value = content or "<empty>"
+					value = (#content > 0 and content) or "empty"
```
Comment (AntoineJT) [inline module_modo.lua:272]: `<empty>` est devenu `empty`, ce qui ne va pas

Comment (SirLynix) [reply module_modo.lua]: This will trigger an error with a nil content, as you check its size before checking if it's valid.

**Comment by SirLynix** [comment]
LGTM, thanks

**Pull request merged**